README.md

```diff
@@ -104,6 +104,30 @@ module.exports = {
 
 NB: In order for webpack to properly resolve scss imports locally, you must use a `~` before the import, like so: `@import "~@edx/brand/paragon/fonts";`
 
+### Running tests
+
+You can run all tests as follows:
+```
+nvm use
+npm run test
```

Comment:
[nit] I would add the additional command `npm run install` to make sure their dependencies are installed if the user attempts to run tests without downloading the packages.